components/Form/AccordionItem/index.js

```diff
@@ -1,35 +1,44 @@
 import styles from './index.module.scss';
+import { useState } from 'react';
 
 const AccordionItem = ({
   children,
   heading,
   id,
   hasSelectedVulnerabilities,
-  hasSelectedAssets
-}) => (
-  <div className="govuk-accordion__section" data-testid="accordion-item">
-    <div
-      className={`govuk-accordion__section-header ${styles['lbh-accordion__section-header']}`}
-    >
-      <h3 className="govuk-accordion__section-heading">
-        <span className="govuk-accordion__section-button" id={id}>
-          {heading}
-        </span>
-        {hasSelectedAssets && (
-          <span className={`${styles['assets-dot']}`}></span>
-        )}
-        {hasSelectedVulnerabilities && (
-          <span
-            hidden={true}
-            className={`${styles['vulnerabilities-dot']}`}
-          ></span>
-        )}
-      </h3>
+  hasSelectedAssets,
+  onClick
+}) => {
+  const [expanded, setExpanded] = useState(false);
+  return (
+    <div className="govuk-accordion__section" data-testid="accordion-item">
+      <div
+        className={`govuk-accordion__section-header ${styles['lbh-accordion__section-header']}`}
+        onClick={() => {
+          onClick(!expanded);
```

Comment:
Maybe we should rename the onClick prop passed in - it obviously does something with the expanded state - eg. handleExpanded()